src/scippneutron/absorption/quadratures.py

```diff
@@ -1,3 +1,17 @@
+"""Quadrature rules for computing integrals on the unit disk
+For each dictionary :pycode:`{x: ..., y: ..., weights: ...}` in this module:
+:math:`\\int_{||(x, y)|| < 1} f(x, y) dxdy \\approx \\sum_i weights[i] f(x[i], y[i])`
```

Comment:
```suggestion
"""Quadrature rules for computing integrals on the unit disk.

For each dictionary ``{x: ..., y: ..., weights: ...}`` in this module:
:math:`\\int_{||(x, y)|| < 1} f(x, y) dxdy \\approx \\sum_i weights[i] f(x[i], y[i])`
```

Empty line to separate brief description from remaining body and render the summary properly.

Double backticks for rendering code.